doc/sphinx/source/changelog.rst

```diff
@@ -3,6 +3,129 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+- Two new recipes have been added:
+
+  - Recipe :ref:`recipe_aod_aeronet_assess.yml <recipe_aod_aeronet_assess>`
+    evaluates model aerosol optical depth (AOD) climatologies against ground
+    based observations from the AeroNET measurement network.
+  - Recipe :ref:`recipe_climate_patterns.yml <recipe_climate_patterns>`
+    generates climate patterns from CMIP6 model datasets.
+
+- The ESACCI-WATERVAPOUR CMORizer now includes daily data and uses the
+  officially released CDR2 data.
+- Support for 5 new datasets have been added:
+
+  - AeroNET
+  - ANU Climate 2.0 Australian data
+  - Australian Gridded Climate Data(AGCD) precipitation
+  - NOAA-ERSST
+  - NSIDC-G02202-sh sea ice fraction
+
+- NEW TREND: First time release manager shout-outs!
+
+  - This is the first ESMValTool release managed by the Met Office! We want to
+    shout this out - and for all future first time release managers to
+    shout-out - to celebrate the growing, thriving ESMValTool community.
+
+This release includes
+
+Bug fixes
+~~~~~~~~~
+
+-  Recipe_ocean_quadmap: Update ATSR to match ESGF name (:pull:`3443`) by :user:`rbeucher`
+-  Fix recipe_bock20jgr_fig_8-10.yml (:pull:`3665`) by :user:`LisaBock`
+
+Documentation
+~~~~~~~~~~~~~
+
+-  Improve release tools and documentation (:pull:`3462`) by :user:`bouweandela`
+-  Fix a typo in the references file (:pull:`3499`) by :user:`bouweandela`
+-  Fix recipe path in ``recipe_perfmetrics.rst`` (:pull:`3532`) by :user:`TomasTorsvik`
+-  Improved description of model evaluation recipes (:pull:`3541`) by :user:`schlunma`
+-  Remove double word in cmorizer documentation (:pull:`3553`) by :user:`bettina-gier`
+-  Fix Codacy badge (:pull:`3558`) by :user:`bouweandela`
+-  Update the release schedule for v2.11.0 (:pull:`3573`) by :user:`ehogan`
+-  Improve the formatting of the recipe documentation template (:pull:`3652`) by :user:`mo-gill`
+-  Add introduction material on the main documentation page (:pull:`3628`) by :user:`bouweandela`
+-  Avoid warning in documentation build (:pull:`3675`) by :user:`bouweandela`
+
+Diagnostics
+~~~~~~~~~~~
+
+-  ``monitor/multi_dataset.py`` improvements: allow data w/o ``timerange`` and improve text formatting (:pull:`3528`) by :user:`schlunma`
+-  Allow datasets without ``project`` in multi_datasets.py (:pull:`3552`) by :user:`schlunma`
+-  Avoid concatenation error in recipe_pcrglobwb.yml (:pull:`3645`) by :user:`bouweandela`
```

Comment:
This one could also be moved to Dependency updates

Reply:
Done in ef8737b84 😊